sdk/core/azure-core-test/src/main/java/com/azure/core/test/implementation/RestProxyTests.java

```diff
@@ -29,10 +29,10 @@
 import com.azure.core.annotation.ServiceInterface;
 import com.azure.core.annotation.UnexpectedResponseExceptionType;
 import com.azure.core.http.policy.HttpLogOptions;
-import com.azure.core.implementation.entities.HttpBinFormDataJSON;
-import com.azure.core.implementation.entities.HttpBinFormDataJSON.PizzaSize;
-import com.azure.core.implementation.entities.HttpBinHeaders;
-import com.azure.core.implementation.entities.HttpBinJSON;
+import com.azure.core.implementation.entity.HttpBinFormDataJSON;
+import com.azure.core.implementation.entity.HttpBinFormDataJSON.PizzaSize;
+import com.azure.core.implementation.entity.HttpBinHeaders;
+import com.azure.core.implementation.entity.HttpBinJSON;
```

Comment:
Sorry to be a nuisance, but please revert this rename. I am in the process of deleting this whole package!